ItemGenerator/WordGenerator.cs

```diff
@@ -23,12 +22,10 @@ public string Generate(WordSequenceItem item, int seed)
                 if (item.Words == null || !item.Words.Override)
                     wordList.AddRange(_globalConfiguration.DefaultWords);
 
-                targetWord = wordList[randomSeed.Next(wordList.Count)];
+                targetWord = wordList[cryptoRandom(0, wordList.Count-1)];

```

Comment:
I'm confused, too. I did compile and test before pushing...but I agree that
should not work. Maybe my commit --amend screwed something up. Honestly I'm
constantly worried I'm doing git wrong. I'll look into that today.
On Aug 9, 2015 6:49 AM, "Pascal" <notifications@github.com> wrote:

> In ItemGenerator/WordGenerator.cs
> <https://github.com/fireout/keepasssequencer/pull/11#discussion_r36586961>
> :
>
> > @@ -23,12 +22,10 @@ public string Generate(WordSequenceItem item, int seed)
> >                  if (item.Words == null || !item.Words.Override)
> >                      wordList.AddRange(_globalConfiguration.DefaultWords);
> >
> > -                targetWord = wordList[randomSeed.Next(wordList.Count)];
> > +                targetWord = wordList[cryptoRandom(0, wordList.Count-1)];
>
> This confuses me, does it compile? (I can not try it myself yet)
> Either missing the .GetRandomInRange or delegate definition, altough if a
> delegate is defined, might as well use it throughout...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fireout/keepasssequencer/pull/11/files#r36586961>.
>